src/runtime_src/core/tools/common/ReportAie.cpp

```diff
@@ -132,6 +132,13 @@ ReportAie::writeReport(const xrt_core::device* /*_pDevice*/,
 
 	if(tile.second.find("dma") != tile.second.not_found()) {
           _output << boost::format("    %s:\n") % "DMA";
+
+          _output << boost::format("        %s:\n") % "FIFO";

```

Comment:
Weird syntax.  For it really doesn't do anything and isn't justified.  Did you mean to use this instead:
```suggestion
          _output << boost::format("%-12s:\n") % "FIFO";

```

Reply:
Fixed. Thanks